extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Anchor.fm integration.
+ *
+ * @since 9.0.0
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\AnchorFm;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Podcast_Helper;
+
+const FEATURE_NAME = 'anchor-fm';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers Anchor.fm integration for the block editor.
+ */
+function register_block() {
+	Blocks::jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'plan_check' => false,
+		)
+	);
+
+	// Register post_meta for connecting Anchor podcasts with posts.
+	register_post_meta(
+		'post',
+		'anchor_podcast',
+		array(
+			'show_in_rest' => true,
+			'single'       => true,
+			'type'         => 'string',
+		)
+	);
+	register_post_meta(
+		'post',
+		'anchor_episode',
+		array(
+			'show_in_rest' => true,
+			'single'       => true,
+			'type'         => 'string',
+		)
+	);
+	register_post_meta(
+		'post',
+		'anchor_spotify_show',
+		array(
+			'show_in_rest' => true,
+			'single'       => true,
+			'type'         => 'string',
+		)
+	);
+}
+
+/**
+ * Checks URL params to determine the Anchor integration action to perform.
+ */
+function process_anchor_params() {
+	$current_screen = \get_current_screen();
+	// TODO: Replace `$current_screen->is_block_editor()` with `wp_should_load_block_editor_scripts_and_styles()` that is introduced in WP 5.6.
```

Comment:
How about adding a `function_exists()` check so we can use this already when possible? It would make it easier to update things in the future.

Reply:
It triggers a fatal error for simple Jetpack sites

![image](https://user-images.githubusercontent.com/77539/101482628-c240ab00-3935-11eb-99f0-cbd203952fa3.png)

Reply:
I have used this exact code to match a different occurance. I always prefer to use exact code in all places for things like this so they are easier to do in bulk when ready for an update.  https://github.com/Automattic/jetpack/blob/b067e4d51685b8ba15e6590ce6b4c881d68e5a38/modules/widget-visibility/widget-conditions.php#L25-L29

If your solution is much more preferred, it should probably be done in all places.

Reply:
> If your solution is much more preferred, it should probably be done in all places.

Yup, I agree. And in that case, I'd leave it for a follow up to keep this PR focused on the relevant changes.

Reply:
Same Fatal for me:

```
PHP Fatal error:  Uncaught Error: Call to undefined function get_current_screen() in /wp-content/plugins/jetpack-dev/extensions/blocks/anchor-fm/anchor-fm.php:62
Stack trace:
#0 /wp-includes/class-wp-hook.php(287): Automattic\Jetpack\Extensions\AnchorFm\process_anchor_params()
#1 /wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#2 /wp-includes/plugin.php(484): WP_Hook->do_action()
#3 /wp-includes/script-loader.php(2188): do_action()
#4 /wp-includes/class-wp-hook.php(287): wp_common_block_scripts_and_styles()
#5 /wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#6 /wp-includes/plugin.php(484): WP_Hook->do_action()
#7 /wp-includes/script-loader.php(2004): do_action()
#8 /wp-includes/class-wp-hook.php(287): wp_enqueue_scripts()
#9 /wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#10 /wp-includes/plugin.php(484): WP_Hook->do_action()
#11 /wp-includes/general-template.php(3005): do_action()
#12 /wp-content/themes/zuki-wpcom/header.php(21): wp_head()
#13 /wp-includes/template.php(730): require_once('/ap...')
#14 /wp-includes/template.php(676): load_template()
#15 /wp-includes/general-template.php(48): locate_template()
#16 /wp-content/themes/zuki-wpcom/archive.php(9): get_header()
#17 /wp-includes/template-loader.php(106): include('/ap...')
#18 /wp-blog-header.php(19): require_once('/ap...')
#19 /index.php(16): require('/ap...')
#20 {main}
  thrown in /wp-content/plugins/jetpack-dev/extensions/blocks/anchor-fm/anchor-fm.php on line 62
```

> I'd leave it for a follow up to keep this PR focused on the relevant changes.

Sounds good to me!